log4j-api/src/main/java/org/apache/logging/log4j/Logger.java

```diff
@@ -1686,6 +1687,14 @@ void fatal(String message, Object p0, Object p1, Object p2, Object p3, Object p4
      */
     <MF extends MessageFactory> MF getMessageFactory();
 
+    /**
+     * Gets the flow message factory used to convert messages into flow messages.
+     *
+     * @return the flow message factory, as an instance of {@link FlowMessageFactory}.
+     * @since 2.20
+     */
+    FlowMessageFactory getFlowMessageFactory();
```

Comment:
Just wanted to say thanks for handling this here as I was also adding this field in 3.x, so that simplified things for me!